### PR TITLE
Added the Contribution Guidelines in README.md

### PR DESCRIPTION
**Pull request by Saikatssd**
It will help the beginners to contribute on this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,3 +197,40 @@ See my _awful_ [commit history](https://github.com/onedr0p/home-ops/commits/main
 ## 🔏 License
 
 See [LICENSE](./LICENSE)
+
+ - # Contribution guidelines
```
Comment (onedr0p) [inline README.md:201]:
```suggestion
## Contribution guidelines 
```

File: README.md [hunk 2]
```diff
@@ -197,3 +197,40 @@ See my _awful_ [commit history](https://github.com/onedr0p/home-ops/commits/main
 ## 🔏 License
 
 See [LICENSE](./LICENSE)
+
+ - # Contribution guidelines 
+
+Welcome to [home-ops]! We're thrilled that you'd like to contribute. Your help is essential for making it better.
+
+## Getting Started
```
Comment (onedr0p) [inline README.md:205]:
```suggestion
### Getting Started
```

**Comment by onedr0p** [comment]
I would rather this being in a separate file CONTRIBUTING.md, thanks!

**Comment by onedr0p** [comment]
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

`.github/CONTRIBUTING.md`

**Comment by Saikatssd** [comment]
Sir I have committed the changes please review it.

**Review by onedr0p — APPROVED** [review]

**Pull request merged**